server/jetty/src/main/java/io/deephaven/server/jetty/JettyConfig.java

```diff
@@ -128,6 +134,15 @@ public int port() {
     @Nullable
     public abstract Boolean http1();
 
+    public abstract OptionalInt http2StreamIdleTimeout();
```

Comment:
Can we make this an `Optional<Duration>` instead? (I was going to suggest OptionalLong to match `#setStreamIdleTimeout(long)`, but think Duration is better.)

Reply:
We can also have a `@Check` annotation to ensure the value makes sense (ie, I'm assuming negative timeouts should not be allowed.)

Reply:
Technically negative values are allowed.

Reply:
I'd still vote no, unless we have a consumer that takes Duration (for example, move to ServerConfig and pass to a matching method there to use it in netty... which doesnt exist, and if it did, it would be long+TimeUnit).

Otherwise we're just building up extra checks to turn a number (as is our convention in every other timeout/duration) into an object, then right back into a number again.

Following jetty (and ServerBuilder), I will change this to a long, and add a `Millis` suffix to the name.